packages/framework/src/Model/Product/Brand/BrandCachedFacade.php

```diff
@@ -4,21 +4,21 @@
 
 namespace Shopsys\FrameworkBundle\Model\Product\Brand;
 
+use Shopsys\FrameworkBundle\Component\LocalCache\LocalCacheFacade;
 use Shopsys\FrameworkBundle\Component\Router\FriendlyUrl\FriendlyUrlFacade;
-use Symfony\Contracts\Service\ResetInterface;
 
-class BrandCachedFacade implements ResetInterface
+class BrandCachedFacade
 {
-    /**
-     * @var array<int, array<int, string>>
-     */
-    protected array $brandUrlsIndexedByBrandIdAndDomainId = [];
+    protected const string BRAND_URL_CACHE_NAMESPACE = 'brandUrlCache';
```

Comment:
You do not use word `Cache` in previous strings, so remove it here too.

Reply:
i dont understand what you meen